app/scripts/lib/fxa-client.js

```diff
@@ -20,14 +20,45 @@ function (FxaClient, $, p, Session, AuthErrors) {
     return $.trim(str);
   }
 
+  // errors from the FxaJSClient must be normalized so that they
+  // are translated and reported to metrics correctly.
+  function wrapClientToNormalizeErrors(client) {
```

Comment:
Nit pic: wrapClientToNormalizeErrors to wrapClientToNormalizedErrors (normalize to normalizeD)